### PR TITLE
REM 330

### PR DESCRIPTION
**Pull request by imtithalhalalo**
Adding batch post from and getting the api and status from the URL

### CODE REVIEW

File: src/pages/batchImports/[resourceId]/index.js [hunk 1]
```diff
@@ -213,6 +213,7 @@ const BatchImports = () => {
           },
           width: 500,
           height: 450,
+          closable: false,
```
Comment (RebeccaFrem02) [inline src/pages/batchImports/[resourceId]/index.js:216]: did you make sure you don't need draggable:flase?

Comment (imtithalhalalo) [reply src/pages/batchImports/[resourceId]/index.js:216]: it can be draggable

File: src/components/Shared/ThreadProgress.js [hunk 2]
```diff
@@ -103,7 +103,7 @@ export const ThreadProgress = ({ recordId, access, window }) => {
   const currentPhaseProgress = data.phases ? (data.currentPhase / data.phases) * 100 : 0
   const completedProgress = data.iterations ? (data.completed / data.iterations) * 100 : 0
 
-  const tasksCompleted = currentPhaseProgress === 100 && completedProgress === 100
+  const tasksCompleted = data.currentPhase === data.phases && data.completed === data.iterations
```
Comment (RebeccaFrem02) [inline src/components/Shared/ThreadProgress.js:106]: could this break anything?

File: src/pages/batchpost/forms/BatchPostForm.js [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+  const today = new Date()
+
+  const { formik } = useForm({
+    initialValues: {
+      startDate: new Date(today.getFullYear(), 0, 1),
+      endDate: today,
+      status: parseInt(status),
+      batchId: 0,
+      plantId: 0
```
Comment (Alinadimmokdad) [inline src/pages/batchpost/forms/BatchPostForm.js:35]: why 0 , not empty strings ?

**Pull request merged**